osidb/models.py

```diff
@@ -1422,6 +1422,21 @@ def trackers_filed(self):
             for affect in self.affects.filter(resolution=Affect.AffectResolution.FIX)
         )
 
+    @property
+    def task_workflow_state(self):
+        """return the current task workflow state name"""
+        return self.task.workflow_state if hasattr(self, "task") else ""
+
+    @property
+    def task_owner(self):
+        """return the current task owner in Jira"""
+        return self.task.owner if hasattr(self, "task") else ""
+
+    @property
+    def task_team_id(self):
+        """return the current team id in Jira"""
+        return self.task.team_id if hasattr(self, "task") else ""
```

Comment:
It would be more natural to me to return `None` when there is really nothing. The empty string is still a valid string so I it is less significant nothing :smile: If there is however reason to return the empty string, I do not have a strong preference.

Reply:
I do not have a strong preference either, I have changed to `None`.